.eslintrc.js

```diff
@@ -0,0 +1,46 @@
+// http://eslint.org/docs/user-guide/configuring
+
+module.exports = {
+  root: true,
+  env: {
+    browser: true,
+    node: true,
+    es6: true,
+    jquery: true,
```

Comment:
No need this on this project right?

Reply:
You right. It seems not needed. I will check what every parameter is for in `env` and `globals`

Reply:
Env is for context, globals is for globals variables.